README.md

```diff
@@ -5,66 +5,117 @@ Library to simplify login process with Infomaniak oauth 2.0 protocol
 ## Install
 
 In the `build.gradle` of your lib/app (not root level), add this instruction :
-```
-implementation 'com.github.Infomaniak:android-login:2.0'
+
+```groovy
+implementation 'com.github.Infomaniak:android-login:2.6.1'
 ```
 
-## Use
+## Use Login
 
 In the `onCreate` method of your Activity/Fragment, instantiate the library :
-```
-		val loginInstance = InfomaniakLogin(
-			context = this,
-			clientId = CLIENT_ID,
-			redirectUri = "$APP_UID:/oauth2redirect")
+
+```kotlin
+val loginInstance = InfomaniakLogin(
+    context = this,
+    clientId = CLIENT_ID,
+    redirectUri = "$APP_UID:/oauth2redirect"
+)
 ```
 
 with these arguments :
+
 - `context` : The current activity/fragment context (will be used to show a Chrometab)
 - `clientId` : The client ID of the app
 - `redirectUri` : The redirection URL after a successful login (in order to handle the codes)
 
 **Once instantiated, you may use this everywhere in your activity/fragment, for example in a `onClick` method :**
 
-```
-		buttonConnect.setOnClickListener {
-			loginInstance.start()
-		}
+```kotlin
+buttonConnect.setOnClickListener {
+    loginInstance.start()
+}
 ```
 
 It'll create a custom chrome tab or launch the browser.
-If needed (to see if an error occured), the `start()` method returns a boolean indicating the success (or not) of the operation.
-
+If needed (to see if an error occurred), the `start()` method returns a boolean indicating the success (or not) of the operation.
 
 **Or you can use the `loginInstance` to create a login from a webview.**
 
+```kotlin
+buttonConnect.setOnClickListener {
+    infomaniakLogin.startWebViewLogin(/*resultLauncher: ActivityResultLauncher<Intent>*/)
+}
 ```
-		buttonConnect.setOnClickListener {
-			infomaniakLogin.startWebViewLogin(2)
-		}
+
+It'll create an activity that contains a webview. The `startWebViewLogin` takes as parameter a resultLauncher to handle the
+activity results.
+
+```kotlin
+private val webViewLoginResultLauncher = registerForActivityResult(StartActivityForResult()) { result ->
+    with(result) {
+        if (resultCode == RESULT_OK) {
+            val authCode = data?.getStringExtra(InfomaniakLogin.CODE_TAG)
+            val translatedError = data?.getStringExtra(InfomaniakLogin.ERROR_TRANSLATED_TAG)
+            when {
+                translatedError?.isNotBlank() == true -> showError(translatedError)
+                authCode?.isNotBlank() == true -> authenticateUser(authCode)
+                else -> showError(getString(RCore.string.anErrorHasOccurred))
+            }
+        }
+    }
+}
 ```
 
-It'll create an activity that contains a webview. The `startWebViewLogin` takes as parameter an integer that will correspond to `requestCode` in `onActivityResult`.
+This allows you to retrieve the answers from the webview, either the `code` for `auth` in case of success, or in case of error, we
```

Comment:
```suggestion
This allows you to retrieve the answers from the WebView, either the `code` for `auth` in case of success, or in case of error, we
```